### ISSUE TITLE
add empty "alt" attributes to "span.facultyImage" > img in "People Listing": 1

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#325]
At present, no "alt" attributes appear on the "facultyImage" ```<img>``` elements that appear in the "People Listing." This is a WCAG A accessibility violation. E.g.:

```html
<span class="facultyImage">
    <img src="http://isaw.nyu.edu/people/faculty/isaw-faculty/roderick-b.-campbell/@@images/Image/mini" />            
</span> 
```

But what to put in the ```alt``` attribute? [W3C guidance](https://www.w3.org/WAI/tutorials/images/decorative/) would have us distinguish between "informative" and "decorative" images, with the latter deserving an empty ```alt``` attribute, effectively muting it in screen readers. We think that staff photos would be informative to a sighted person, but not informative to a person experiencing the site through a screen reader; therefore, we will treat these images in the "People Listing" as decorative.

Thus, we should modify the appropriate template to produce:

```html
<span class="facultyImage">
    <img alt="" src="http://isaw.nyu.edu/people/faculty/isaw-faculty/roderick-b.-campbell/@@images/Image/mini" />            
</span> 
```

## Related epics:

 - #324: accessibility for the "People Listing"
 - #257: ensure every image has useful "alt" text

**Comment by jessesnyder** [comment]
Fixed the same issue when a Person appears in the `folder_listing` view. No extra charge.
